### DISCUSSION TITLE
Where would I put an extra "X-…" HTML header to be sent with my Nextcloud pages?

### DISCUSSION THREAD

**Discussion by Moonbase59** [nextcloud/all-in-one#2164] [Questions] [answered]
Say, for some reason, I wanted to put an extra "X-…" header in the HTML headers for my Nextcloud AIO  pages. (Assume a "[GNU Terry Pratchett](http://www.gnuterrypratchett.com/)" as an example.)

Where would I put that?

Is there maybe already a config option I overlooked?

**Comment by szaimen** [comment, marked as answer]
Hi, for this I would recommend [to run AIO behind a reverse proxy](https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md). By doing so you can then also add your own custom header in the reverse proxy config.
